src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -931,6 +931,8 @@ def load_arguments(self, _):
             is_preview=True,
             arg_type=get_enum_type(health_probe_modes),
         )
+        c.argument("if_match")
+        c.argument("if_none_match")
         # virtual machines
```

Comment:
Should we support these for `with self.argument_context("aks upgrade") as c:` too? I _think_ upgrade is just PUT MC?

Reply:
and/or PUT AP*

Reply:
Yes, you're right, thanks for catching that. Upgrade should be a PUT unless it's upgrading node image version in which case it is a POST and we do not support etags for POST.